tox.ini

```diff
@@ -5,6 +5,8 @@ envlist =
     py27-django17
     py27-django18
     py27-django19
+    py35-django18
+    py35-django19
```

Comment:
Urgh - we should add django 1.10 and 1.11 here as well

Reply:
I'll open up another PR after this one if that's okay with you :) I think it belongs in a different PR anyway

Reply:
Yeah sure.